bindings/python/CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ INCLUDE_DIRECTORIES(${PYTHON_INCLUDE_PATH}
 	${CMAKE_SOURCE_DIR}/include
 	${CMAKE_CURRENT_SOURCE_DIR})
 
-SET(CMAKE_SWIG_FLAGS "")
+if(ENABLE_TOOLS)
+	set(CMAKE_SWIG_FLAGS -DCOMMUNICATION)
```

Comment:
Do we need to have "-DCOMMUNICATION" or just -DCOMMUNICATION  is fine?

Reply:
For one flag the quotes are not necessary.